dataproc_jupyter_plugin/handlers.py

```diff
@@ -183,6 +193,30 @@ def post(self):
         else:
             self.finish({'config' : ERROR_MESSAGE + 'failed'})
 
+
+class UrlHandler(APIHandler):
+    @tornado.web.authenticated
+    def get(self):
+        url = {}
+        dataproc_url = gcp_service_url('dataproc')
+        compute_url = gcp_service_url('compute', default_url='https://compute.googleapis.com/compute/v1')
+        metastore_url = gcp_service_url('metastore')
+        cloudkms_url = gcp_service_url('cloudkms')
+        cloudresourcemanager_url = gcp_service_url('cloudresourcemanager')
+        datacatalog_url = gcp_service_url('datacatalog')
+        storage_url = gcp_service_url('storage', default_url='https://storage.googleapis.com/storage/v1/')
```

Comment:
I'm a bit confused as to when we have default urls with paths and when we don't.  Lets be consistant in how we do it across all endpoints?

Reply:
This comes from `gcloud`... the defaults it uses differs depending on the API.